docs/book/src/reference/glossary.md

```diff
@@ -199,11 +199,13 @@ The infrastructure that backs a [Machine Resource](#user-content-machine), typic
 # T
 ---
 
-### Target cluster
+### Target management cluster
```

Comment:
I've seen people use the word "target cluster" to talk about the workload cluster, a.k.a. not the management cluster. What does "target management cluster" mean? The definition below is a little unclear to me, it seems like it can be both the management cluster and the workload cluster depending on the situation...

Reply:
Originally "Target" was introduced to differentiate the pivoting use case specifically, rather than trying to use the existing management cluster/workload cluster type terminology which doesn't really hold up for pivoting use cases (where one could pivot to or from a workload cluster, or in some use cases to/from might not even be a workload cluster).

The idea was to use Source/Target to talk about the pivot workflow instead. This clarification was my attempt to try and disambiguate the usage a bit. Since target only applies to pivoting, then it is only applicable to management clusters and not any other type of cluster we discuss.

I'm open to any other way we can disambiguate these concepts a bit more though, because we haven't been overly successful to date :joy:

Reply:
Could we just remove the terminology "target" from all docs then? And instead just refer to "management" and "workload" clusters everywhere?

Reply:
sgtm

Reply:
IMO we can remove this definition from the glossary, but I defer this to @detiber

Reply:
done!